rustls/src/verify.rs

```diff
@@ -590,13 +590,24 @@ impl ClientCertVerifier for AllowAnyAuthenticatedClient {
         now: SystemTime,
     ) -> Result<ClientCertVerified, Error> {
         let (cert, chain, trustroots) = prepare(end_entity, intermediates, &self.roots)?;
+        let crls = self
+            .crls
+            .iter()
+            .map(|crl_der| webpki::CertRevocationList::from_der(crl_der))
+            .collect::<Vec<Result<CertRevocationList, webpki::Error>>>()
+            .into_iter()
```

Comment:
I think you can collect into `Result<Vec<_>, _>` directly (without the intermediate collection step).

Reply:
Good call, thanks. Fixed up.